src/redundantaudioencoder/RedundantAudioEncoder.ts

```diff
@@ -236,12 +237,12 @@ export default class RedundantAudioEncoder {
     frame: RTCEncodedAudioFrame,
     controller: TransformStreamDefaultController
   ): void {
-    // const frameMetadata = frame.getMetadata();
```

Comment:
Is this a miss from previous PR? Does not seem related with the issue this PR tries to fix.

Reply:
Yeah this was missed from the previous CR

Reply:
@dinmin-amzn Curious why a unit test did not catch this. Maybe we can add one?

Reply:
Yes, let me add a unit test for this

Reply:
Updated the existing unit tests to better capture this case.